src/containers/footer-links/tests/footer-links.spec.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import FooterLinks from '../footer-links';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
You can remove adapter enzyme and shallow.

Reply:
Cause after reconfiguration those imports are redunant, same for all other tests